src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js

```diff
@@ -387,6 +389,16 @@ define(['config', 'widget-groups'], function(Config, WidgetGroups) {
                 this.numberOfBankAccounts--;
                 this.updateBankAccountAddIcon(this.numberOfBankAccounts);
             }, this);
+
+            this.sandbox.on('sulu.router.navigate', this.cleanUp.bind(this));
+        },
+
+        /**
+         * Does some cleanup with aura components
+         */
+        cleanUp: function() {
+            // stop contact form before leaving
+            this.sandbox.stop(constants.editFormSelector);
```

Comment:
Maybe that's also an input for @marcelmoosbrugger, because the components should be stopped on a tab change IMO, so that something like this is not necessary.